app/assets/images/dark-slider-bean.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="44" height="50" viewBox="0 0 44 50" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<title>dark slider bean</title>
+<desc>Created using Figma</desc>
+<g id="Canvas" transform="translate(-10236 -271)">
+<g id="dark slider bean">
+<g id="Vector">
+<use xlink:href="#path0_fill" transform="matrix(0.905499 -0.424348 0.420892 0.907111 10229 281.074)" fill="#33160E"/>
+</g>
+<g id="Vector">
+<use xlink:href="#path1_fill" transform="matrix(0.905499 -0.424348 0.420892 0.907111 10246 280.771)" fill="#602B20"/>
+</g>
+</g>
+</g>
+<defs>
+<path id="path0_fill" d="M 20.118 51.0177C 31.2288 51.0177 40.236 39.597 40.236 25.5089C 40.236 11.4207 31.2288 0 20.118 0C 9.00713 0 0 11.4207 0 25.5089C 0 39.597 9.00713 51.0177 20.118 51.0177Z"/>
+<path id="path1_fill" d="M 3.99323 0.595162C 2.28399 2.63854 1.07484 5.04973 0.461207 7.63843C -0.152428 10.2271 -0.15375 12.9225 0.457345 15.5118C 1.89609 21.148 10.2847 28.8736 2.29844 33.7324C 0.298839 34.9471 2.12775 38.1053 4.13955 36.8785C 8.26069 34.3762 9.93109 29.0679 8.87032 24.5006C 8.17534 21.4517 5.95626 19.3988 4.7248 16.6293C 3.78125 14.4305 3.45306 12.0181 3.77508 9.6484C 4.0971 7.27868 5.05729 5.04009 6.55371 3.17034C 8.10218 1.40902 5.52952 -1.17831 3.96885 0.595162L 3.99323 0.595162Z"/>
+</defs>
+</svg>
```

Comment:
Any images/SVG's for individual Roasts should be handled by the CMS. 

Is this going to be animated? Or just a photo?